<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RemoteControlHttp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/31314.png</normaloff>:/images/31314.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="CustomWidgetsQt::LogWindow2" name="logWindow" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>38</width>
     <height>38</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStartServer"/>
   <addaction name="actionStopServer"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionStartServer">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text">
    <string>Запуск</string>
   </property>
   <property name="toolTip">
    <string>Запуск</string>
   </property>
  </action>
  <action name="actionStopServer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Остановка</string>
   </property>
   <property name="toolTip">
    <string>Остановка</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomWidgetsQt::LogWindow2</class>
   <extends>QWidget</extends>
   <header>CustomWidgets/LogWindow2/logwindow2.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
